sonar-python-plugin/src/test/java/org/sonar/plugins/python/PythonSensorTest.java

```diff
@@ -319,7 +319,7 @@ public void cross_files_issues_only_one_file_analyzed() {
     // "mod.py" created but not added to context
     InputFile modFile = createInputFile("mod.py");
     PythonIndexer pythonIndexer = pythonIndexer(Arrays.asList(mainFile, modFile));
-    sensor(CUSTOM_RULES, pythonIndexer, null).execute(context);
+    sensor(null, pythonIndexer, analysisWarning).execute(context);
```

Comment:
why did you change CUSTOM_RULES -> null ?

Reply:
I wanted to have a unit test with a `PythonIndexer` and without a `PythonCustomRuleRepository` to reflect the new constructor.